messaging.rst

```diff
@@ -100,26 +100,31 @@ Fields
 |  balance_proof       | BalanceProof  | Balance proof for this transfer                            |
 +----------------------+---------------+------------------------------------------------------------+
 
-Mediated Transfer
+Locked Transfer
 -----------------
 
 Cancellable and expirable :term:`transfer`. Sent by a node when a transfer is being initiated, this message adds a new lock to the corresponding merkle tree of the sending participant node.
```

Comment:
Is the `Locked Transfer` message a `transfer`? 
`transfer = A movement of tokens from a Sender to a Receiver.` from the terminology. It does not say anything about the finality - wether it is `non-refundable` or `non-cancellable`

Reply:
> Is the Locked Transfer message a transfer?

If we define the transfer as the *movement of tokens*, then the `Locked Transfer` message alone does not qualify. A transfer would be initialized by the `LockedTransfer` message and finalized by the `Unlock` message.

>  It does not say anything about the finality - wether it is non-refundable or non-cancellable

The transfer initiated with locked with the message `LockedTransfer` is cancellable, and refundable. 

The transfer can be cancelled if the initiator doesn't send the first secret reveal message, and wait for it to expire.

The transfer may be refunded to a mediator by another, if another message with a lock with the same amount and secrethash is sent in reverse.

Reply:
If we use transfer to the movement of tokens, then we would have two types of transfers:

- Direct transfers, implemented by a single direct transfer message. The transfer is non-expirable and non-refundable.
- Locked Transfers. The transfer is implemented by the messages LockedTransfer, RevealSecret, SecretRequest, Unlock. The *transfer* is expirable, cancellable. The message LockedTransfer is refundable.